packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -318,6 +342,38 @@ class Auth {
         )
     }
 
+    /**
+     * This is a wrapper method for ShopperCustomer API registerCustomer endpoint.
+     *
+     */
+    async register(
+        body: Argument<ShopperCustomers<ApiClientConfigParams>['registerCustomer']>['body']
```

Comment:
```suggestion
        body: ShopperCustomersTypes.CustomerRegistration
```
And you knew this! (L359) 😝